src/localisation/CMakeLists.txt

```diff
@@ -1,5 +1,214 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(localisation)
 
-# This package is currently just used to house the EKF launch file. 
-# If we end up adding more in-house localisation things, they should go here
+## Find catkin macros and libraries
```

Comment:
In general, we try and remove unused parts of `CMakeLists.txt` files, as the default is pretty big. For an example of this, see `src/decision/CMakeLists.txt`